edgemesh/cmd/conf/chassis.yaml

```diff
@@ -0,0 +1 @@
+ssl:
```

Comment:
please add a blank line at the end of the file.
Applicable to similar places

Reply:
done